diff --git a/src/xrt/state_trackers/oxr/oxr_api_space.c b/src/xrt/state_trackers/oxr/oxr_api_space.c
index 8526b6644..896507685 100644
--- a/src/xrt/state_trackers/oxr/oxr_api_space.c
+++ b/src/xrt/state_trackers/oxr/oxr_api_space.c
@@ -232,7 +232,7 @@ oxr_xrLocateSpace(XrSpace space, XrSpace baseSpace, XrTime time, XrSpaceLocation
        OXR_VERIFY_SPACE_NOT_NULL(&log, baseSpace, baseSpc);
        OXR_VERIFY_ARG_TYPE_AND_NOT_NULL(&log, location, XR_TYPE_SPACE_LOCATION);
 
-       if (time <= (XrTime)0) {
+       if (time < (XrTime)0) {
                return oxr_error(&log, XR_ERROR_TIME_INVALID, "(time == %" PRIi64 ") is not a valid time.", time);
        }
 
diff --git a/src/xrt/state_trackers/oxr/oxr_api_swapchain.c b/src/xrt/state_trackers/oxr/oxr_api_swapchain.c
index 20f371312..c697f27e8 100644
--- a/src/xrt/state_trackers/oxr/oxr_api_swapchain.c
+++ b/src/xrt/state_trackers/oxr/oxr_api_swapchain.c
@@ -98,8 +98,7 @@ oxr_xrCreateSwapchain(XrSession session, const XrSwapchainCreateInfo *createInfo
        }
 
        if ((createInfo->usageFlags & ~flags) != 0) {
-               return oxr_error(&log, XR_ERROR_VALIDATION_FAILURE,
-                                "(createInfo->usageFlags == 0x%04" PRIx64 ") contains invalid flags",
+               oxr_warn(&log, "(createInfo->usageFlags == 0x%04" PRIx64 ") contains invalid flags",
                                 createInfo->usageFlags);
        }
        bool format_supported = false;